doc/newsboat.txt

```diff
@@ -784,6 +784,73 @@ To enable this mode, the following configuration needs to be done:
 "opml" must be specified as source for the feed URLs, and the URLs of the OPML
 file needs to be specified, too.
 
+Inoreader Support
+~~~~~~~~~~~~~~~~~
+
+http://inoreader.com/[Inoreader] is another successor to Google Reader.
+Newsboat provides functionality to use The Inoreader as its
+backend: people can use Inoreader to manage their subscriptions, and in
+addition, use newsboat to download and read articles. Newsboat will keep
+the information which articles have already been read synchronized with
+Inoreader, so that users usually won't see articles more than once. In addition, it
+will only ever download unread articles from Inoreader.
+
+In order to use Inoreader support, you first need to configure the proper URL source:
+
+	urls-source "inoreader"
+
+In addition, newsboat needs to know your Inoreader username and password
+so that it can authenticate with Inoreader.
+Note that this is NOT your login with your Google or Facebook account. If you
+use one of those to login to Inoreader, you have to create a username and
+password in Inoreader Preferences > Profile
```

Comment:
Missing period at the end. Perhaps the "Preferences > Profile" bit should be in italics to make it clearer these are names of the menu items, not just general directions.